src/windows/MediaProxy.js

```diff
@@ -203,20 +234,64 @@ module.exports = {
         var id = args[0];
         var thisM = Media.get(id);
 
-        var normalizedSrc = thisM.src.replace(/\//g, '\\');
+        var success = function () {
+            Media.onStatus(id, Media.MEDIA_STATE, Media.MEDIA_STOPPED);
+            win();
+        };
+
+        var error = function (reason) {
+            Media.onStatus(id, Media.MEDIA_ERROR, reason);
+            lose(reason);
+        };
+
+        var fsTypes = {
+            PERSISTENT: 'PERSISTENT',
+            TEMPORARY: 'TEMPORARY'
+        };
+        var fsType;
+
+        var normalizedSrc = thisM.src;
+        normalizedSrc = setTemporaryFsByDefault(normalizedSrc);
+
+        if (normalizedSrc.indexOf('ms-appdata:///local/') === 0) {
```

Comment:
Is this component going to allow for interoperable code among platforms?  I just want to ensure that the developer isn't expected to utter "ms-appdata:///local/" - instead they're getting that from some service (or we're converting into that from some other argument) so that the developer can just write "persistent" or "temporary" and it saves.